securing_apps/topics/saml/java/jboss-adapter/jboss-adapter-samesite-setting.adoc

```diff
@@ -0,0 +1,17 @@
+[[_saml-jboss-adapter-samesite-setting]]
+===== Setting SameSite value for JSESSIONID cookie
+
+SAML Post binding might be broken in the latest versions of browsers due to changes to the SameSite default value. To
+preserve full functionality of the SAML adapter, it is recommended to set the `SameSite` value to `None` for
+`JSESSIONID` cookie created by your container. Not doing so may result in resetting container's session with each
+request to {project_name}.
+
+To set the `SameSite` value to `None` for `JSESSIONID` cookie in Wildfly/EAP add file `undertow-handlers.conf`, with
+following content, to the `WEB-INF` directory of your application.
+
+ samesite-cookie(mode=None, cookie-pattern=JSESSIONID)
+
+The support for this configuration is available in Wildfly from version 19.1.0 and in EAP from versions 7.3.2.GA and
```

Comment:
Neither EAP 7.3.2 nor 7.2.9 is available. Refer the reader to EAP documentation for determining when this can be used, or don't reference EAP at all.